tap_shopify/streams/order_refunds.py

```diff
@@ -48,6 +49,9 @@ def sync(self):
             refund_dict = refund.to_dict()
             replication_value = strptime_to_utc(refund_dict[self.replication_key])
             if replication_value >= bookmark:
+                for transaction_dict in refund_dict["transactions"]:
```

Comment:
Is the transactions key guaranteed? Might be good to change it to `.get("transactions",[])` for weird data safety.